build.gradle

```diff
@@ -54,25 +54,6 @@ allprojects {
 apply plugin: 'com.palantir.external-publish-jar'
```

Comment:
We also need to move applying this plugin to `refreshable/build.gradle`. This plugin is responsible for publishing the jars produced by the project and currently its just publishing the now empty root project and not the sub-project.

Reply:
Ah of course. I have pushed a change. I have run a local maven publish to test that this works correctly now.